src/gamepad/standard-gamepad-axis.vala

```diff
@@ -0,0 +1,33 @@
+// This file is part of GNOME Games. License: GPLv3
+
+// This file is part of GNOME Games. License: GPLv3
+
+public enum LibGamepad.StandardGamepadAxis {
+	/**
+	 * The horizontal axis of the left stick.
+	 *
+	 * Negative is left, positive is right.
+	 */
+	LEFT_X,
+
+	/**
+	 * The vertical axis of the left stick.
+	 *
+	 * Negative is up, positive is down.
+	 */
+	LEFT_Y,
+
+	/**
+	 * The horizontal axis of the right stick.
+	 *
+	 * Negative is left, positive is right.
+	 */
+	RIGHT_X,
+
+	/**
+	 * The vertical axis of the right stick.
+	 *
+	 * Negative is up, positive is down.
+	 */
+	RIGHT_Y,
+}
```

Comment:
We will probably need an "invalid" or "unknown" value to parse the strings correctly (see a previous comment).